src/igvfd/schemas/donor.json

```diff
@@ -36,7 +36,7 @@
     ],
     "properties": {
         "schema_version": {
```

Comment:
if in the changelog you claim it to be "### Schema version 3" why here is it hidden?

Reply:
It was version 3 in my previous commit. Jennifer requested to change it to match other schemas- see her comment above.

Reply:
@kalina i think the changelog should match the schema, otherwise it is confusing
please change to version 3 (to match changelog)

Reply:
In other abstract parent classes, we don't have a default schema version. I'm confused which way we should go with?

Reply:
Perhaps the Donor changelog could be changed to be like the "mixins" changelog, where we do not mention any schema version, and only write the date that the change was implemented? https://github.com/IGVF-DACC/igvfd/blob/dev/src/igvfd/schemas/changelogs/mixins.md